client/dashboard/sites/settings-repositories/connect-repository.tsx

```
@@ -21,13 +21,29 @@ export default function ConnectRepository() {
 		navigate( { to: '/sites/$siteSlug/settings/repositories' } );
 	};
 
+	const breadcrumbs = [
+		{
+			label: __( 'Settings' ),
+			path: `/sites/${ siteSlug }/settings`,
+		},
+		{
+			label: __( 'Repositories' ),
+			path: `/sites/${ siteSlug }/settings/repositories`,
+		},
+		{
+			label: __( 'Connect' ),
+			path: `/sites/${ siteSlug }/settings/repositories/connect`,
+		},
```

This looks a bit strange because we are technically not displaying this but we need to add the current page here. Breadcrumbs component by default hides the last item. There is a prop that allows us to display the last item but then it treats it as current item meaning it does not allow the link to bring you elsewhere.